**********result from 9:1 split of train and validation**********
InvalidArgumentError: indices[4022] = 16339 is not in [0, 16320)
	 [[Node: ROI_5/GatherV2_22 = GatherV2[Taxis=DT_INT32, Tindices=DT_INT32, Tparams=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"](ROI_5/strided_slice_46, ROI_5/strided_slice_47, training_8/SGD/gradients/roi_align_classifier_5/concat_grad/mod)]]


**********result from 10:1 of fullset vs. validation**********
InvalidArgumentError: indices[4464] = 16336 is not in [0, 16320)
	 [[Node: ROI_5/GatherV2_22 = GatherV2[Taxis=DT_INT32, Tindices=DT_INT32, Tparams=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"](ROI_5/strided_slice_46, ROI_5/strided_slice_47, training_9/SGD/gradients/roi_align_classifier_5/concat_grad/mod)]]


**********result from 9:1 split of train and validation rerun**********
InvalidArgumentError: indices[5859] = 16336 is not in [0, 16320)
	 [[Node: ROI_5/GatherV2_20 = GatherV2[Taxis=DT_INT32, Tindices=DT_INT32, Tparams=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"](ROI_5/strided_slice_42, ROI_5/strided_slice_43, training_10/SGD/gradients/roi_align_classifier_5/concat_grad/mod)]]

